CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.44.101](https://github.com/brave/brave-browser/releases/tag/v1.44.101)
+
+ - Added .zil TLD support for Unstoppable Domains. ([#24621](https://github.com/brave/brave-browser/issues/24621))
+ - Added "Prevent fingerprinting via language settings" under the "Brave Shields & privacy" settings. ([#24621](https://github.com/brave/brave-browser/issues/23919))  
+ - Added Solana account creation dialog in Brave Wallet under the "Send" tab if Solana account doesn't exist. ([#24463](https://github.com/brave/brave-browser/issues/24463))  
+ - Updated Brave Wallet to create a Solana account by default when creating or restoring a wallet. ([#25432](https://github.com/brave/brave-browser/issues/25432)) 
```

Comment:
probably need to mention that its also setting as default

Reply:
Done 👍